modular_skyrat/modules/ashwalkers/code/effects/ash_rune.dm

```diff
@@ -0,0 +1,69 @@
+/obj/effect/ash_rune
+	name = "ash rune"
+	desc = "A remnant of a civilization that was once powerful enough to harness strange energy for transmutations."
+	icon = 'modular_skyrat/modules/ashwalkers/icons/ash_ritual.dmi'
+	icon_state = "rune"
+
+	/// the list of rituals
+	var/list/rituals = list()
```

Comment:
have this be a static list or a global

Reply:
can't be static because it gets changed. We discussed my reason for not making it global.